requirements.txt

```diff
@@ -54,7 +55,17 @@ notify2==0.3.1; sys_platform != 'win32' and sys_platform != 'darwin'
 # Uncomment line below or manually install after installing requirements.
 # pygobject>=3.10.2; sys_platform != 'win32' and sys_platform != 'darwin'
 
+# PyYAML does not have binary packages for Mac Users
+PyYAML==6.0.1 --no-binary=PyYAML; sys_platform == 'darwin'
+PyYAML==6.0.1; sys_platform != 'darwin'
```

Comment:
We can remove this special `darwin` stuff here as this is only for run-from-source users, for who `pip` will get the correct version. The `--no-binary` in `build_release` will fix it for the release.